project/src/common.js

```diff
@@ -19,4 +21,43 @@ const getDate = function (comment) {
   };
 };
 
-export { getStarRating, getDate };
+const sortByDefault = function (someOffers) {
+  return someOffers;
+};
+
+const sortByPriceToHigh = function (someOffers) {
+  const sortedOffers = [...someOffers].sort((someOfferA, someOfferB) => someOfferA.price - someOfferB.price);
+  return sortedOffers;
+};
+
+const sortByPriceToLow = function (someOffers) {
+  const sortedOffers = [...someOffers].sort((someOfferA, someOfferB) => someOfferB.price - someOfferA.price);
+  return sortedOffers;
+};
+
+const sortByRatingFromTop = function (someOffers) {
+  const sortedOffers = [...someOffers].sort((someOfferA, someOfferB) => someOfferB.rating - someOfferA.rating);
+  return sortedOffers;
+};
+
+const sortBySortType = function (someCurrentOffers, someType) {
+  switch (someType) {
+    case SortType.POPULAR: {
+      return sortByDefault(someCurrentOffers);
+    }
+    case SortType.TO_HIGHT: {
+      return sortByPriceToHigh(someCurrentOffers);
+    }
+    case SortType.TO_LOW: {
+      return sortByPriceToLow(someCurrentOffers);
+    }
+    case SortType.TOP_RATE: {
+      return sortByRatingFromTop(someCurrentOffers);
+    }
+    default: {
+      return sortByDefault(someCurrentOffers);
+    }
```

Comment:
```suggestion
    default: {
      return someCurrentOffers;
    }
```
зачем вообще сортировать?